activities/RAVLT/RAVLT_schema.jsonld

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": [
+    "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/contexts/generic.jsonld",
+    "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/RAVLT/RAVLT_context.jsonld"
+  ],
+  "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Activity.jsonld",
+  "@id": "RAVLT_schema",
+  "skos:prefLabel": "Rey Auditory Verbal Learning Test",
+  "skos:altLabel": "RAVLT",
+  "schema:description": "This activity is a demo of the Rey Auditory Verbal Learning Test cognitive task.",
+  "schema:schemaVersion": "0.0.1",
+  "schema:version": "0.0.1",
+  "preamble": "Please follow the instructions for the task.",
+  "branchLogic": {},
+  "scoringLogic": {},
+  "variableMap": [],
```

Comment:
can you add the variable map here? I know it's not being used, but I think we'll want this to be a required value eventually